package/Dockerfile

```diff
@@ -20,23 +24,46 @@ COPY bin/longhorn-instance-manager /usr/local/bin/
 
 COPY package/engine-manager /usr/local/bin/
 
+# Build gperftools
+# 1. Just use tcmalloc lib from gperftools installation
+# 2. tcmalloc also supports arm64 platform, but we only focus on amd64 first currently.
+#    arm64 will be supported after more testing.
+RUN if [ ${ARCH} == "amd64" ]; then \
+        cd /usr/src && \
+        git clone https://github.com/gperftools/gperftools.git&& \
+        cd gperftools && \
+        git checkout db6d94d9893fdd8c74028b5113cd9185be1e3111 && \
+        ./autogen.sh && \
+        ./configure && \
+        make && \
+        make install; \
+    fi
```

Comment:
Is `rm -rf /usr/src/gperftools` missed here?
Similar below

Reply:
Yeah, we can delete it.

Reply:
Addressed. Thank you.